OutOfSchool/OutOfSchool.BusinessLogic/Services/TagService.cs

```diff
@@ -0,0 +1,145 @@
+﻿using AutoMapper;
+using Microsoft.Extensions.Localization;
+using OutOfSchool.BusinessLogic.Enums;
+using OutOfSchool.BusinessLogic.Models.Tag;
+
+namespace OutOfSchool.BusinessLogic.Services;
+
+/// <summary>
+/// Implements the interface with CRUD functionality for for SocialGroup entity.
+/// </summary>
+public class TagService : ITagService
+{
+    private readonly IEntityRepository<long, Tag> repository;
+    private readonly ILogger<TagService> logger;
+    private readonly IStringLocalizer<SharedResource> localizer;
+    private readonly IMapper mapper;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="TagService"/> class.
+    /// </summary>
+    /// <param name="repository">Repository.</param>
+    /// <param name="logger">Logger.</param>
+    /// <param name="localizer">Localizer.</param>
+    /// <param name="mapper">Mapper.</param>
+    public TagService(
+        IEntityRepository<long, Tag> repository,
+        ILogger<TagService> logger,
+        IStringLocalizer<SharedResource> localizer,
+        IMapper mapper)
+    {
+        this.localizer = localizer ?? throw new ArgumentNullException(nameof(localizer));
+        this.repository = repository ?? throw new ArgumentNullException(nameof(repository));
+        this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        this.mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
+    }
+
+    /// <inheritdoc/>
+    public async Task<IEnumerable<TagDto>> GetAll(LocalizationType localization = LocalizationType.Ua)
+    {
+        logger.LogInformation($"Getting all Tags, {localization} localization, started.");
```

Comment:
(Logging level) Related for all methods. 
Please consider changing `LogInformation` to `LogTrace`/`Debug`. 

At least for me it looks like debugging information that could be hidden until some issue occured.